octodns/provider/ultra.py

```diff
@@ -444,7 +453,12 @@ def _apply_Delete(self, change):
                existing._type == self.RECORDS_TO_TYPE[record['rrtype']]:
                 zone_name = self._remove_prefix(existing.fqdn,
                                                 existing.name + '.')
-                path = '/v2/zones/{}/rrsets/{}/{}'.format(zone_name,
+
+                # UltraDNS treats the `APEXALIAS` type as the octodns `ALIAS`.
+                if existing._type == "ALIAS":
+                    existing._type = "APEXALIAS"
+
+                path = '/v2/zones/{}/rrsets/{}/{    }'.format(zone_name,
                                                           existing._type,
```

Comment:
Best not to change the `Record` type here .  I believe the following should be sufficient. 

```suggestion
                existing_type = existing._type
                if existing_type == "ALIAS":
                    existing_type = "APEXALIAS"

                path = '/v2/zones/{}/rrsets/{}/{    }'.format(zone_name,
                                                          existing_type,
```